src/web/components/form/TagListDisplay.tsx

```diff
@@ -0,0 +1,46 @@
+/* SPDX-FileCopyrightText: 2025 Greenbone AG
+ *
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+import styled from 'styled-components';
+import tagStyles from 'web/components/form/tagStyles';
+
+interface TagListDisplayProps {
+  values: string[];
+  color?: keyof typeof tagStyles;
+}
+
+const Tag = styled.span<{bg: string; color: string}>`
+  background-color: ${({bg}) => bg};
+  color: ${({color}) => color};
+  padding: 2px 8px;
+  border-radius: 12px;
+  font-size: 0.85em;
+  font-weight: 500;
+  margin-right: 6px;
+`;
+
+const TagList = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+`;
```

Comment:
Suggestion: I think instead of using a `<span>` on L36, we can use `gap` and avoid the `span`.
```suggestion
  gap: 8px;
```